framework/web/UrlRule.php

```diff
@@ -108,6 +113,11 @@ class UrlRule extends Object implements UrlRuleInterface
      * @since 2.0.7
      */
     protected $placeholders = [];
+    /**
+     * @var string|null status of URL creation after last [[createUrl()]] call.
+     * @since 2.0.12
+     */
+    public $createStatus;
```

Comment:
I don't think this property should be public. I see no reason to modify this property from outside of the class.

`getCreateStatus()` method may be moved here from stub.